package.json

```diff
@@ -85,9 +91,21 @@
       ]
     },
     "angular-mock": "global:angular.mock",
-    "diff-match-patch": "diff_match_patch",
-    "dom-text-mapper": "DomTextMapper",
-    "dom-text-matcher": "DomTextMatcher",
+    "dom-anchor-fragment": "domAnchorFragment",
+    "dom-anchor-text-position": {
+      "depends": [
+        "dom-seek:seek"
+      ],
+      "exports": "domAnchorTextPosition"
```

Comment:
I find this awkward. I should maybe rename the files. Even if the packages and repos are `dom-anchor-foo` maybe if I make the files `FooAnchor.js` then babel will make more sensible UMD wrappers that export `FooAnchor` to the global scope instead of `domAnchorFoo`.